src/xmipp/libraries/reconstruction/resolution_fsc.h

```diff
@@ -0,0 +1,65 @@
+/***************************************************************************
+ *
+ * Authors:    Jose Luis Vilas (jlvilas@cnb.csic.es)
+ *
+ * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+ * 02111-1307  USA
+ *
+ *  All comments concerning this program package may be sent to the
+ *  e-mail address 'xmipp@cnb.csic.es'
+ ***************************************************************************/
+
+#ifndef _PROG_RES_FSC
+#define _PROG_RES_FSC
+
+#include <core/xmipp_program.h>
+#include <core/xmipp_fftw.h>
```

Comment:
it seems to me that fftw and metadata_extension includes are not necessary in this header.
use core/xmipp_filename.h and forward declare MultidimArray 
```
template<typename T>
class MultidimArray;
```